backend/core/forms.py

```diff
@@ -22,3 +21,10 @@ def clean(self):
             raise ValidationError("Choose one, and only one, filter to apply")
 
         return data
+
+
+class BulkAddUsersForm(forms.Form):
+    email_addresses = forms.CharField(
+        label="Email addresses",
+        widget=forms.Textarea(attrs={"rows": 15, "style": "display: flex; width:100%"}),
```

Comment:
Can you add some extra info about the format of the input. How must the user know how to seperate the email addresses? 

It would also be good to tap into Django's own form validation stuff.

What should the error message be if an email address isn't found?

Reply:
Django's form api has a lo of functionality. You are using it just to draw a form but it should be used all the way. 

Ideally, we should be able to say `form.cleaned_data["email_addresses"]` to get the email addresses of the users. Or, better yet, `form.cleaned_data["users"]`

Reply:
I couldn't figure out how to get it working with `form.cleaned_data["users"]`.